controller/api/Deliveries.php

```diff
@@ -0,0 +1,199 @@
+     *                             "report": {
+     *                                 {
+     *                                     "type": "info",
+     *                                     "message": "Running task http://sample/first.rdf#i1111111111111111"
+     *                                 }
+     *                             }
+     *                         }
+     *                     },
+     *                     "version": "3.4.0-sprint131"
+     *                 }
+     *             )
+     *         ),
+     *     ),
+     *     @OA\Response(
+     *         response="400",
+     *         description="Delivery publishing bad request",
+     *         @OA\MediaType(
+     *             mediaType="application/json",
+     *             @OA\Schema(
+     *                 @OA\Property(
+     *                     property="success",
+     *                     type="boolean",
+     *                     description="`false` on failure, `true` on success",
+     *                 ),
+     *                 @OA\Property(
+     *                     property="errorCode",
+     *                     type="string",
+     *                     description="Exception error code",
+     *                 ),
+     *                 @OA\Property(
+     *                     property="errorMsg",
+     *                     type="string",
+     *                     description="Exception message",
+     *                 ),
+     *                 @OA\Property(
+     *                     property="version",
+     *                     type="string",
+     *                 ),
+     *                 example= {
+     *                     "success": false,
+     *                     "errorCode": 0,
+     *                     "errorMsg": "Delivery resource with URI 'http://BAD/URI.rdf#i1' does not exist.",
+     *                     "version": "3.4.0-sprint131"
+     *                 }
+     *             )
+     *         ),
+     *     ),
+     * )
+     * @OA\Schema(
+     *     schema="TaskLog",
+     *     type="object",
+     *     @OA\Property(
+     *         property="id",
+     *         type="string",
+     *         description="ID"
+     *     ),
+     *     @OA\Property(
+     *         property="status",
+     *         type="string",
+     *         description="Status"
+     *     ),
+     *     @OA\Property(
+     *         property="report",
+     *         type="array",
+     *         @OA\Items(
+     *             ref="#/components/schemas/TaskReport",
+     *         )
+     *     ),
+     * )
+     * @OA\Schema(
+     *     schema="TaskReport",
+     *     type="object",
+     *     @OA\Property(
+     *         property="type",
+     *         type="string",
+     *         description="Type"
+     *     ),
+     *     @OA\Property(
+     *         property="message",
+     *         type="string",
+     *         description="Message"
+     *     ),
+     * )
+     */
+    public function publish()
```

Comment:
As an option we can add method parameter `ServerRequest $request` and it will be injected automatically.

Reply:
Updated